.gitignore

```diff
@@ -2,7 +2,7 @@
 node_modules
 
 # Built files
-adapters
+/adapters
```

Comment:
Why was this changed?

Reply:
I think thats how git works. The built files are in the root which should be ignored. With just adapters, it will ignore adapters in the src folder too. So any changes to netlify will also not reflect. The files that exist before adding .gitignore will be there.

Reply:
Ah gotcha, forgot about the adapters folder in src. Good catch!